### PR TITLE
VZ-7059 Override validation bug allows multiple overrides in one list value for some components

### PR DESCRIPTION
**Pull request by knelson7**

### CODE REVIEW

File: platform-operator/controllers/verrazzano/component/jaeger/operator/jaeger_operator_component.go [hunk 1]
```diff
@@ -152,23 +152,44 @@ func (c jaegerOperatorComponent) ValidateInstall(vz *vzapi.Verrazzano) error {
...
 	}
+	if err := c.HelmComponent.ValidateInstallV1Beta1(&convertedVZ); err != nil {
+		return err
+	}
 	return c.validateJaegerOperator(&convertedVZ)
 }
 
 // ValidateUpdate validates if the update operation of the Verrazzano CR is valid or not.
 func (c jaegerOperatorComponent) ValidateUpdate(old *vzapi.Verrazzano, new *vzapi.Verrazzano) error {
+
```
Comment (anders-swanson) [inline platform-operator/controllers/verrazzano/component/jaeger/operator/jaeger_operator_component.go:163]: it's a very minor thing, but can you remove the extra whitespace at the top of every function body?

File: platform-operator/controllers/verrazzano/component/rancherbackup/rancher_backup_component_test.go [hunk 2]
```diff
@@ -182,11 +187,90 @@ func TestPostUninstall(t *testing.T) {
...
+
+func TestValidateMethods(t *testing.T) {
+	tests := []struct {
+		name    string
+		vz      *v1alpha1.Verrazzano
+		wantErr bool
+	}{
+		{
+			name: "singleOverride",
+			vz: &v1alpha1.Verrazzano{
```
Comment (anders-swanson) [inline platform-operator/controllers/verrazzano/component/rancherbackup/rancher_backup_component_test.go:207]: this is just a suggestion, but you can reduce amount of boilerplate by creating reusable functions that create CRs for the test cases. it is not required.

File: platform-operator/controllers/verrazzano/component/jaeger/operator/jaeger_operator_component_test.go [hunk 3]
```diff
@@ -986,3 +1019,73 @@ func getVZCRWithDefaultJaegerOverride(jaegerCROverride string) *vzapi.Verrazzano
 		},
 	}
 }
+
+func TestValidateBetaInstall(t *testing.T) {
```
Comment (desagar) [inline platform-operator/controllers/verrazzano/component/jaeger/operator/jaeger_operator_component_test.go:1023]: this test now does install and update validation so should be renamed accordingly.

**Comment by desagar** [comment]
I would suggest starting your PR title and your commit message at the time of merge with the JIRA number VZ-7059 instead of Ksnelson/vz 7059. If you do that, you will get automatic updates in the JIRA when you merge, which is very useful for anyone trying to map commits to jiras (for example at release time).

**Review by anders-swanson — APPROVED** [review]
lgtm

**Pull request merged**